app/server/models/user.py

```diff
@@ -260,17 +260,19 @@ def location(self):
 
     @location.setter
     def location(self, location):
-        from server.utils.location import async_set_user_gps_from_location
 
         self._location = location
```

Comment:
location now doesn't automagically call the async lookup as we don't want it to always happen. Now call `attempt_update_gps_location` below